components/location/popup.jsx

```diff
@@ -0,0 +1,319 @@
+/* eslint-disable react/prop-types */
+import React, { useContext } from "react";
+
+import { LanguageContext } from "@contexts/translator";
+
+///////////
+// returns a an array of urls as strings if there is a match otherwise null
+// e.g. ['https://google.com']
+///////////
+function extractUrls(item) {
+  // web URL regex source: https://stackoverflow.com/questions/6927719/url-regex-does-not-work-in-javascript
+  const host_pattern = /(?:[a-z][\w-]+:(?:\/{1,3}|[a-z0-9%])|www\d{0,3}[.]|[a-z0-9.-]+[.][a-z]{2,4}\/)/;
+  const path_pattern = /(?:[^\s()<>]+|\(([^\s()<>]+|(\([^\s()<>]+\)))*\))+/;
+  const end_pattern = /(?:\(([^\s()<>]+|(\([^\s()<>]+\)))*\)|[^\s`!()[\]{};:'".,<>?«»“”‘’])/;
+  const web_pattern =
+    host_pattern.source + path_pattern.source + end_pattern.source;
+  // phone regexp adapted from: https://stackoverflow.com/a/16699507
+  const phone_pattern = /^(\+\d{1,2}\s)?\(?\d{3}\)?[\s.-]?\d{3}[\s.-]?\d{4}$/;
+  // email regexp adapted from https://www.regular-expressions.info/email.html
+  const email_pattern = /[\w.%+-]+@[\w.]+\.[a-z]{2,}/;
+  const url_pattern = new RegExp(
+    /\b/.source + // all URLs start after a word boundary
+      "(" +
+      web_pattern +
+      "|" +
+      phone_pattern.source +
+      "|" +
+      email_pattern.source +
+      ")",
+    "gi" // find all occurrences, case-insensitive
+  );
+
+  return item.match(url_pattern);
+}
+
+function parseLineBreaks(value) {
+  return value.replace(/\n/g, "<br />");
+}
+
+const PopupSection = ({ content, id }) => {
+  const { getTranslation } = useContext(LanguageContext);
+
+  return (
+    <div className="p row">
+      <p
+        // data-translation-id="${_.snakeCase(id)}"
+        className="key"
+      >
+        {getTranslation(id)}
+      </p>
+      {/** Kanad Note: I had to change this from a p tag to a div tag
+       * since it was causing issues when doing a ReactDOMServer render */}
+      <div className="value">{content}</div>
+    </div>
+  );
+};
+
+const NeedsMoney = ({ location }) => {
+  const { getTranslation } = useContext(LanguageContext);
+
+  let link = "";
+  if (location.seekingMoneyURL && location.seekingMoneyURL !== "") {
+    link = (
+      <a
+        data-translation-id="seeking_money_link"
+        href="${location.seekingMoneyURL}"
+        target="_blank"
+        // onclick="captureOutboundLink('${location.seekingMoneyURL}', 'donation')"
```

Comment:
Same question as above - does this no longer track then?

Reply:
Same answer as above!
>Yeah I haven't reimplemented the Google Analytics tracking at all, but I'm sure we could add some onClick handler function that we pass down to here! Per our discussion I think I'd like to rip out GA entirely and go with [Netlify Analytics](https://www.netlify.com/products/analytics/) instead, but happy to discuss that further!